RequestManagementSystem/Service/ReqProxyHandler.py

```diff
@@ -117,9 +117,8 @@ def sweeper( cls ):
             continue
           gLogger.info( "sweeper: successfully put request '%s' @ ReqManager" % cachedName )
           os.unlink( cachedFile )
-        except Exception, error:
+        except Exception as error:
           gLogger.exception( "sweeper: hit by exception %s" % str( error ) )
```

Comment:
This is wrong!
should be
gLogger.exception( "sweeper: hit by exception", lException = error)
since str(error) is incorrect. Alternative: use repr(error)